docs/doc-config.js

```diff
@@ -6,7 +6,9 @@ module.exports = {
      * Refer to doc by relative path without extension, i.e.: 'maps/drilldown'
      */
     unlisted: [
-        'export-module/legacy-export-servers'
+        'export-module/legacy-export-servers',
+        'stock/compare',
+        'stock/cumulative-sum'
```

Comment:
Are you sure you want them as unlisted? It means they are not reachable from the navigation menu...

Reply:
Of course...  corrected